### PR TITLE
Updated release notes for Target v3.1.0

### PR DESCRIPTION
**Pull request by swarna04**
Updated release notes with details on the fix for MOB-14886/ TNT-40395

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Please note that this release introduces breaking changes. For more details, see
 
 * Fixed an issue where the click notification was not being sent to Adobe Target for a retrieved mbox location upon the `clickedLocation` API call.
 
+* Added support for sending click conversion A4T payload to Adobe Analytics for A4T enabled Target activities upon the `clickedLocation` API call.
```
Comment (calebk1) [inline release-notes/README.md:49]: "... for sending the click conversion..." 

"... for A4T-enabled Target activities when the `clickLocation` API is called."

File: using-mobile-extensions/adobe-target/adobe-target-release-notes.md [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 * Fixed an issue where the click notification was not being sent to Adobe Target for a retrieved mbox location upon the `clickedLocation` API call.
 
+* Added support for sending click conversion A4T payload to Adobe Analytics for A4T enabled Target activities upon the `clickedLocation` API call.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-target/adobe-target-release-notes.md:23]: "... for sending the click conversion..." 

"... for A4T-enabled Target activities when the `clickLocation` API is called."

**Review by calebk1 — COMMENTED** [review]
Left some feeeback!

**Comment by swarna04** [comment]
Feedback's incorporated @calebk1. Thanks for your review. Let me know if the pr looks good to merge.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**